libs/monorepo-utils/src/circleci.ts

```diff
@@ -21,6 +21,7 @@ function generateTestJobForNodeJsPackage(
   }
 
   const hasPlaywrightTests = existsSync(`${pkg.path}/playwright.config.ts`);
+  const hasSnapshotTests = existsSync(`${pkg.path}/src/__image_snapshots__`);
```

Comment:
Ohhh, now I see how you added it across libs. You could make this logic more robust to catch all the `__image_snapshots__` folders. But that seems low pri compared to VSAP work if its a hassle - can you make a ticket for us to track?

Reply:
Yup! https://github.com/orgs/votingworks/projects/23/views/15?pane=issue&itemId=62790720